site/src/main/resources/org/typelevel/sbt/helium/default.template.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="${?laika.site.metadata.language}">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="generator" content="Laika 0.18.1 + Helium Theme" />
+    <title>${cursor.currentDocument.title}</title>
+    @:for(laika.site.metadata.authors)
+      <meta name="author" content="${_}"/>
+    @:@
+    @:for(laika.site.metadata.description)
+      <meta name="description" content="${_}"/>
+    @:@
+    @:for(helium.favIcons)
+      <link rel="icon" @:attribute(sizes, _.sizes) @:attribute(type, _.type) @:attribute(href, _.target)/>
+    @:@
+    @:for(helium.webFonts)
+      <link rel="stylesheet" href="${_}">
+    @:@
+    @:linkCSS { paths = ${helium.site.includeCSS} }
+    @:linkJS { paths = ${helium.site.includeJS} }
+    @:heliumInitVersions
+    @:heliumInitPreview(container)
+    <script> /* for avoiding page load transitions */ </script>
+  </head>
+
+  <body>
+
+    <header id="top-bar">
+
+      <div class="row">
+        <a id="nav-icon">
+          @:icon(navigationMenu)
+        </a>
+        @:for(laika.versions)
+        <div id="version-menu-container">
+          <a id="version-menu-toggle" class="text-link drop-down-toggle" href="#">
+            @:if(laika.versioned)
+            ${helium.topBar.versionPrefix} ${laika.versions.currentVersion.displayValue}
+            @:else
+            Documentation
+            @:@
+          </a>
+          <nav id="version-menu">
+            <ul id="version-list" class="nav-list">
+            </ul>
+          </nav>
+        </div>
+        @:@
+      </div>
+  
+      ${?helium.topBar.home}
+      
+      ${?helium.topBar.links}
+      
+    </header>
+
+    <nav id="sidebar">
+
+      ${?helium.topBar.phoneLinks}
+      
+      @:navigationTree {
+        entries = [ 
+          { target = "/", excludeRoot = true, excludeSections = true, depth = 2 } 
+        ]
+      }
+      
+      <ul class="nav-list">
+        <li class="level1 nav-header">Related Projects</li>
+        <li class="level2"><a href="https://typelevel.org/cats/">cats</a></li>
+        <li class="level2"><a href="https://typelevel.org/cats-effect/">cats-effect</a></li>
+        <li class="level2"><a href="https://fs2.io/">fs2</a></li>
+        <li class="level2"><a href="https://scalacheck.org/">scalacheck</a></li>
+      </ul>
```

Comment:
@danicheg any thoughts on this one?

Reply:
Should be that bar be configurable? It's easy to imagine that a particular project could be not related to scalacheck/fs2/etc.

Reply:
Yes, for sure, I was thinking about this too. I started by grabbing the "pinned" projects from the Typelevel org page.

We can make it configurable, or change it from "related" to something else? Or just list every project there? 😆 

If we make it configurable, then maybe we need add some helper method so its easy to add TL libraries there without checking the name, url etc.

Reply:
I was thinking we can use a `Seq[(String, URL)` and provide something like this:
https://github.com/sbt/librarymanagement/blob/67988135152f88afa9134fcfc357c0901e700f40/core/src/main/scala/sbt/librarymanagement/License.scala#L10

Reply:
Sure. More type-safety is always preferable.